### ISSUE TITLE
 Microsoft 365 learning pathways has a configuration issue, Ask your administrator for assistance.

### ISSUE THREAD

**Issue by sajid070** [pnp/custom-learning-office-365#484]
>**PROVISIONING NOTE**: If you are having issues with the SharePoint look book while trying to provision M365 learning pathways please do not post that issue to this repo as we cannot help you here. Please post all questions and issues for the SharePoint look book to [SharePoint/sp-provisioning-service GitHub Issues List](https://github.com/SharePoint/sp-provisioning-service/issues).

>**WEB PART NOT LOADING**: If you are getting the error: "Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance". Please review the [Diagnosing Pre-Check](https://github.com/pnp/custom-learning-office-365/blob/main/installation/DiagnosisPreCheck.md) guidance before submitting your bug.

## Describe the bug

When loading the O365 

### To Reproduce

>Steps to reproduce the behavior

### Expected behavior

>A clear and concise description of what you expected to happen.

### Screenshots

>If applicable, add screenshots to help explain your problem.
blob:https://teams.microsoft.com/f6c1b481-97e0-4158-b75e-d35c0efeef4c
### Learning Pathways version number

>Please share the version number of learning pathways in your tenant app catalog.

### Additional context

>Add any other context about the problem here.

Access to XMLHttpRequest at 'https://securemail.qfc.qa/cgi/tm?code=9db94fd151911135' (redirected from 
'https://graph.microsoft.com/v1.0/me/photo/$value') from origin 'https://webshell.suite.office.com' has 
been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not 
allowed for a preflight request.
securemail.qfc.qa/cgi/tm?code=9db94fd151911135:1 Failed to load resource: net::ERR_FAILED
M365LP/SitePages/Work-remotely.aspx:1 Access to XMLHttpRequest at 'https://waconatm.officeapps.live.com/apc/trans.gif?815a42481e876463eb40f176dedbea22' 
from origin 'https://qfcqa.sharepoint.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is
 present on the requested resource.
waconatm.officeapps.live.com/apc/trans.gif?815a42481e876463eb40f176dedbea22:1 Failed to load resource: net::ERR_FAILED

**Comment by jnewgard-thrive** [comment]
Hi @sajid070,

Unfortunately, you have provided no description of the issue or screenshot. Please complete filling out the issue template with the requested details:

- Steps To Reproduce
- Expected Behavior
- Screenshot
- Learning Pathways Version

Additionally, it is best if you can provide a screenshot or export of the entire console log from your browser dev tools.

**Comment by tcrobbins4** [comment]
This is the same issue in other threads here.  After provisioning Learning Pathways and SharePoint Success site successfully, you try to go to the SSS Administration and get this same message everyone is asking about.  
![image](https://user-images.githubusercontent.com/32391514/113582559-83c2b080-95f6-11eb-9b77-479a89a6ef23.png)

**Comment by jnewgard-thrive** [comment]
Hi @tcrobbins4,

This is most likely do to skipping step 11 of the provisioning instructions. The SharePoint Success Site was provisioned, but the matching Content Pack was not.

[Provisioning SharePoint Success Site](https://docs.microsoft.com/en-us/sharepoint/provision-sss#provision-the-sharepoint-success-site-to-your-tenant-from-the-learning-pathways-administrative-page)

![image](https://user-images.githubusercontent.com/80363676/113584581-4d7e3480-95e8-11eb-9c70-601d1c8c00af.png)

If you do not see the SharePoint Success Site as below in the Learning Pathways Administrative Page, it needs to be provisioned.

![image](https://user-images.githubusercontent.com/80363676/113584627-5cfd7d80-95e8-11eb-8281-3d51c9c1c661.png)

If you encounter further issues, please open a new issue here as we cannot confirm that your issue matches the original post.

[Create New Issue](https://github.com/pnp/custom-learning-office-365/issues/new/choose)

**Comment by tcrobbins4** [comment]
Thanks.  I will try that in a demo tenant.  I've heard that these MLP and SSS will not work in GCC so I might be fighting a losing battle.

**Comment by tcrobbins4** [comment]
Ok.  Got it to work.  Many thanks for that help.  I will try it next in my prod GCC tenant and see if I can get it to work....

**Comment by jellis-thrive** [comment]
user advised issue is resolved with the information provided
Close ticket
